src/locales/pt_br/translation.json

```diff
@@ -1052,9 +1062,12 @@
       "amountSent": "Quantidade enviada",
       "to": "Para",
       "amountReceived": "Quantidade recebida",
+      "transactionFee":"Taxa de transação",
+      "gasFee": "Gás"
```

Comment:
I think needs a comma here @pietro-maximoff

Reply:
yes

Reply:
Added here https://github.com/DistributedCollective/Sovryn-frontend/pull/1910 so you can merge. Thanks